src/matlab2tikz.m

```diff
@@ -3289,9 +3295,10 @@ case size(cData,2) % specific y-data is given
         [xDataPlot, yDataPlot] = deal(xData, yData);
     end
 
-    drawOpts = opts_print(m2t, drawOptions, ',');
-    [m2t, table ] = makeTable(m2t, '', xDataPlot, '', yDataPlot);
-    str = sprintf('\\addplot[%s] plot table[row sep=crcr] {%s};\n', drawOpts, table);
+    [m2t, table, tabOpts] = makeTable(m2t, '', xDataPlot, '', yDataPlot);
+    str = sprintf('\\addplot[%s] plot table[%s] {%s};\n', ...
+                 opts_print(m2t, drawOptions, ','), ...
+                 opts_print(m2t, tabOpts, ','), table);
```

Comment:
I completely agree with this comment, but let's do that in a different PR. I have some simplifications in my private repo and some other small changes (e.g. use of `opts_…` everywhere), where that seems a better fit since those changes are going to touch the whole code base.